set.mm

```diff
@@ -461739,6 +461767,24 @@ its graph has a given second element (that is, function value).
       EIKUHUIUOWIWK $.
   $}
 
+  $( Elementhood in a open integer interval, except its lower bound.
+     (Contributed by Thierry Arnoux, 1-Jan-2024.) $)
```

Comment:
"in an open ..."

Reply:
`( M ... N )` is not an open interval. It's a "finite set of sequential integers", or a "finite sequence of integers " or a "finite interval of integers".